webapp/src/selectors/index.ts

```diff
@@ -13,17 +13,14 @@ const getPluginState = (state) => state['plugins-' + PluginId] || {};
 
 export const getPluginServerRoute = (state) => {
     const config = getConfig(state);
+    const baseUrl = new URL(config?.SiteURL ?? window.location.href);
```

Comment:
I would prefer `||` here since I think `SiteURL` may be an empty string if not provided in the config

Reply:
Can we make this change @Kshitij-Katiyar ?

Reply:
Sure